src/components/CheckBoxs.jsx

```diff
@@ -1,64 +1,64 @@
 import React from "react";
 import { Text, StyleSheet, View, Pressable } from "react-native";
-import { Ionicons } from '@expo/vector-icons';
+import { Ionicons } from "@expo/vector-icons";
 
-export const CheckBox = () => {
+export const CheckBox = ({ title, subtitle }) => {
   const [checked, onChange] = React.useState(false);
 
-function onCheckmarkPress() {
-  onChange(!checked);
-}
+  function onCheckmarkPress() {
+    onChange(!checked);
+  }
```

Comment:
I suggest using the previous state if you want to use the current state value. Something like `prevState => !prevState`

Check out:
https://reactjs.org/docs/hooks-reference.html#functional-updates